packages/server/db/init-mongo.js

```diff
@@ -0,0 +1,15 @@
+print('Start #################################################################');
+
+cardzdb = db.getSiblingDB('cardzdb');
+
+cardzdb.createUser({
+    user: 'appadmin',
+    pwd: 'appadminpassword',
```

Comment:
even if it is used just for local I prefer to use environment variables as much as possible and to no have hardcoded login information in the code 😉

Reply:
Indeed, even if it's local, I guess it's better to keep it in the .env (like the other credentials that are already in it)

Reply:
Actually, I removed the creation of user - at least until we really start using the db, we won't be needing anything different from the admin one